assets/src/edit-story/components/inspector/prepublish/prepublishChecklistProvider.js

```diff
@@ -108,12 +149,27 @@ function PrepublishChecklistProvider({ children }) {
     }, [])
   );
 
+  const focusChecklistTab = useCallback(() => {
+    dispatch(PPC_CHECKPOINT_ACTION.ON_PUBLISH_CLICKED);
+    setIsChecklistReviewRequested(true);
+  }, [setIsChecklistReviewRequested]);
+
+  // Use this when a published story gets turned back to a draft.
+  const resetReviewDialogTrigger = useCallback(() => {
+    setIsChecklistReviewRequested(false);
+  }, []);
```

Comment:
Nit: I don't think this function needs to have `trigger` in the name since this function is unaware of the triggers. I think `resetReviewDialog` would be fine too? 🤔 

TBH it's all good to me though 👍